db/migrate/20160311163203_change_data_type_for_column_vendor_in_vms_table_to_citext.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDataTypeForColumnVendorInVmsTableToCitext < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :vendor, :citext
```

Comment:
@carbonin @yrudman I just want to confirm that this would require the extension be installed, including on external postgresql databases not residing on manageiq appliances, right?